perf/perf_c2c.py

```diff
@@ -63,24 +62,13 @@ def setUp(self):
         else:
             self.cancel('perf c2c is not available')
 
-        # Creating temporary file to collect the perf.data
-        self.temp_file = tempfile.NamedTemporaryFile().name
-
         # Getting the parameters from yaml file
         self.record = self.params.get('record_method', default='')
         self.report = self.params.get('report_method', default='')
 
         # Clear the dmesg, by that we can capture the delta at the end of the
         # test.
-        process.run("dmesg -C", sudo=True)
-
-    def verify_dmesg(self):
-        self.whiteboard = process.system_output("dmesg").decode("utf-8")
-        pattern = ['WARNING: CPU:', 'Oops',
-                   'Segfault', 'soft lockup', 'Unable to handle']
-        for fail_pattern in pattern:
-            if 'fail_pattern' in self.whiteboard:
-                self.fail("Test Failed : %s in dmesg" % fail_pattern)
```

Comment:
Can you move dmesg related changes to a separate commit as these are not related to -k or --vmlinux options.